Failed to install netlifyctl using go get

```
Step 20/22 : RUN go get github.com/netlify/netlifyctl
 ---> Running in 746bece28fc4
[91m# github.com/netlify/netlifyctl/commands/init
go/src/github.com/netlify/netlifyctl/commands/init/git.go:25:34: undefined: models.RepoSetup
go/src/github.com/netlify/netlifyctl/commands/init/github.go:88:62: undefined: models.RepoSetup
go/src/github.com/netlify/netlifyctl/commands/init/github.go:95:10: undefined: models.RepoSetup
go/src/github.com/netlify/netlifyctl/commands/init/gitlab.go:66:62: undefined: models.RepoSetup
go/src/github.com/netlify/netlifyctl/commands/init/gitlab.go:73:10: undefined: models.RepoSetup
go/src/github.com/netlify/netlifyctl/commands/init/manual.go:37:61: undefined: models.RepoSetup
go/src/github.com/netlify/netlifyctl/commands/init/manual.go:40:10: undefined: models.RepoSetup
[0mThe command '/bin/sh -c go get github.com/netlify/netlifyctl' returned a non-zero code: 2
```

Used to work a couple of months ago.
Go version: 1.9.2

--- a/ui/task_tracker.go
+++ b/ui/task_tracker.go
@@ -37,7 +37,11 @@ func (t *TaskTracker) Start(msg string) {
 }
 
 func (t *TaskTracker) Step(msg string) {
-	t.s.Prefix = msg
+	if t.isTerm() {
+		t.s.Prefix = msg
+	} else {
+		fmt.Println(msg)
+	}
 }
 
 func (t *TaskTracker) Success(msg string) {

